clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ImageController.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.controllers;
+
+import com.netflix.spinnaker.clouddriver.model.ImageProvider;
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import com.netflix.spinnaker.kork.web.exceptions.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@RestController
+@RequestMapping("/images")
+public class ImageController {
+
+  @Autowired
+  List<ImageProvider> imageProviders;
+
+  @RequestMapping(value = "/{provider}/{imageId}", method = RequestMethod.GET)
+  Artifact getImage(@PathVariable String provider, @PathVariable String imageId) {
```

Comment:
I don't think Artifact is the right thing here. We should be returning a native representation of the image, rather than jamming it into a class meant for passing artifact information along in triggers / executions.

Reply:
Thanks for the feedback!
Oh, I thought that Artifact was used in a more broader context :) I'll have a look and do the change as soon as possible.

Reply:
In order to use the image native representation, I've did a small refactor that can be found in here: https://github.com/spinnaker/clouddriver/pull/2373
`Image` does not exists at the core level. It does in the openstack module with a `TODO` to move to core - so I just executed the TODO :)